features/ui/src/test/java/org/opennms/alec/rest/ALECRestImplTest.java

```diff
@@ -38,12 +44,6 @@
 import org.opennms.alec.engine.dbscan.AlarmInSpaceTimeDistanceMeasure;
 import org.opennms.alec.engine.dbscan.DBScanEngine;
 import org.opennms.alec.engine.dbscan.DBScanEngineFactory;
-import org.opennms.alec.jackson.Agreement;
-import org.opennms.alec.jackson.AgreementImpl;
-import org.opennms.alec.jackson.Configuration;
-import org.opennms.alec.jackson.EngineParameter;
-import org.opennms.alec.jackson.EngineParameterImpl;
-import org.opennms.alec.jackson.KeyEnum;
```

Comment:
someday we'll make IDE templates for everyone so we stop playing the import dance :D

(not your fault, just complaining)